src/Crowdfunding/N3O.Umbraco.Crowdfunding/Content/CrowdfundingPageContent.cs

```diff
@@ -1,9 +1,22 @@
 ﻿using N3O.Umbraco.Content;
+using N3O.Umbraco.Extensions;
+using N3O.Umbraco.Giving.Crowdfunding.Lookups;
+using N3O.Umbraco.Lookups;
+using System.Linq;
+using Umbraco.Extensions;
+using static N3O.Umbraco.Crowdfunding.CrowdfundingConstants;
 
 namespace N3O.Umbraco.Crowdfunding.Content;
 
 public class CrowdfundingPageContent : UmbracoContent<CrowdfundingPageContent> {
     public CrowdfundingCampaignContent Campaign => GetPickedAs(x => x.Campaign);
-    public bool SetupComplete => GetValue(x => x.SetupComplete);
+    public CrowdfundingPageStatus PageStatus => GetPageStatus();
     public CrowdfundingTeamContent Team => GetPickedAs(x => x.Team);
+    public string PageTitle => GetPickedAs(x => x.PageTitle);
+
+    private CrowdfundingPageStatus GetPageStatus() {
+        var status = Content().Value<string>(CrowdfundingPage.Properties.PageStatus);
+        
+        return StaticLookups.GetAll<CrowdfundingPageStatus>().SingleOrDefault(x => x.Name.EqualsInvariant(status));
```

Comment:
@talhamalik4025 Are we storing the page ID or the page name here?

Reply:
Also I think add a method GetStaticLookupAs() similar to GetPickedAs() as a method to do this without the need for a custom method here.